lib/core/src/Cardano/Wallet/TokenMetadata.hs

```diff
@@ -182,21 +186,18 @@ newtype BatchResponse = BatchResponse
 data SubjectProperties = SubjectProperties
     { subject :: Subject
     , owner :: Maybe Signature
-    -- TODO: use Data.SOP.NP and parameterize type by property names
-    -- Name and description are required, both others may be missing the
-    -- response.
     , properties :: ( Property "name"
-                    -- , (PropertyValue "acronym", [Signature])
```

Comment:
Looking at https://input-output.atlassian.net/wiki/spaces/AD/pages/2137456733/Token+Metadata+Well-Known+Properties now this seems right, modulo:
1. "unit" is here disabled
2. It seems that spec fundamentally is different from the current server implementation — property values are not wrapped in { "value": ., "anSignatures": [] } there… 🤔

Reply:
Yes